Library/Formula/nuxeo.rb

```diff
@@ -0,0 +1,74 @@
+class Nuxeo < Formula
+  desc "Nuxeo Platform - Enterprise Content Management"
+  homepage "https://nuxeo.github.io/"
+  url "http://community.nuxeo.com/static/releases/nuxeo-7.10/nuxeo-cap-7.10-tomcat.zip"
+  version "7.10"
+  sha256 "9e95fb4e3dd0fdac49395ae8b429c6cc7db3c621ecd4d0dff74ea706a2aba723"
+
+  depends_on "poppler" => :recommended
+  depends_on "pdftohtml" => :optional
+  depends_on "imagemagick"
+  depends_on "ghostscript"
+  depends_on "ufraw"
+  depends_on "libwpd"
+  depends_on "ffmpeg" => :optional
+
+  def install
+    libexec.install Dir["#{buildpath}/*"]
+
+    bin.mkpath
+    libexec.mkpath
+
+    # link binary
+    bin.install_symlink "#{libexec}/bin/nuxeoctl"
+    # Inreplace Homebrew' paths to be independant from env variables
+    inreplace "#{libexec}/bin/nuxeoctl" do |s|
+      s.gsub! /\${NUXEO_HOME:=.*$/, '${NUXEO_HOME:=$(cd "$(dirname `PRG=$0; while [ -h "$PRG" ] ;do ls=\\\\`ls -ld "$PRG"\\\\`; link=\\\\`expr "$ls" : ".*-> \\\\(.*\)\$"\\\\`; if expr "$link" : "/.*" &> /dev/null; then PRG="$link"; else PRG="\\\\`dirname "$PRG"\\\\`/$link"; fi; done; echo $PRG`)"/..; pwd -P)}'
+      s.gsub! /\${NUXEO_CONF:=.*$/, "${NUXEO_CONF:=\"#{etc}/nuxeo.conf\"}"
+    end
+
+    # Initiatialize var dir
+    var_nuxeo = "#{var}/nuxeo"
+    unless Dir.exist?(var_nuxeo)
+      mkdir_p(var_nuxeo)
+    end
+
+    # Move data folder if not exists yet
+    unless Dir.exist?("#{var_nuxeo}/data")
+      mv(libexec/"nxserver/data", "#{var_nuxeo}/data")
+    end
```

Comment:
I mean check for a file you're installing in that directory in case the user decides to delete the contents of that directory and expects a `reinstall` to put those files back.